opennms-webapp/src/main/webapp/about/index.jsp

```diff
@@ -135,6 +135,10 @@
     <th>Time-Series Strategy:</th>
     <td><%=TimeSeries.getTimeseriesStrategy().getDescr()%></td>
   </tr>
+  <tr>
+    <th>Message Broker Strategy:</th>
+    <td>RPC: <%=System.getProperty("org.opennms.core.ipc.rpc.strategy", "ActiveMQ")%>; Sink: <%=System.getProperty("org.opennms.core.ipc.sink.strategy", "ActiveMQ")%></td>
```

Comment:
This looks good enough but there is also `GRPC` as IPC strategy for both Sink and RPC which we set through system property `org.opennms.core.ipc.strategy` which takes precedence over  `Sink` and `RPC` specific strategies.

Reply:
> This looks good enough but there is also `GRPC` as IPC strategy for both Sink and RPC which we set through system property `org.opennms.core.ipc.strategy`. This takes precedence over `Sink` and `RPC` specific strategies.

Great catch. Thanks, I'll update the PR to cover this case as well.